digital_land/commands.py

```diff
@@ -233,8 +242,10 @@ def pipeline_run(
         ),
         HarmonisePhase(
             field_datatype_map=specification.get_field_datatype_map(),
+            specification=specification,
```

Comment:
Do we need the spec in here? (we may do but the field_datatype_map above is already passed in)